src/metricscollector/config/config.go

```diff
@@ -4,17 +4,16 @@ import (
 	"fmt"
 	"io"
 	"strings"
-	"time"
 
 	"github.com/cloudfoundry-incubator/candiedyaml"
 
 	"cf"
 )
 
 const (
-	DefaultLoggingLevel                  = "info"
-	DefaultRefreshInterval time.Duration = 60 * time.Second
-	DefaultPollInterval    time.Duration = 30 * time.Second
+	DefaultLoggingLevel    = "info"
+	DefaultRefreshInterval = 60
```

Comment:
Yes, for testing.
For the end user, they will only see seconds because we will generate the config.